app/assets/stylesheets/application.scss

```diff
@@ -88,4 +88,16 @@ main {
 .currency-symbol {
   font-size: 25px;
   vertical-align: super;
-}
\ No newline at end of file
+}
+
+.pac-container {
+  z-index: 20;
+}
+
+.modal {
+  z-index: 20;   
```

Comment:
this will affect _all_ modals... are you trying to affect _one_ particular modal? Same with the `.modal-backdrop`. Why is the override needed?